charts/rpe-send-letter-service/values.yaml

```diff
@@ -1,37 +1,58 @@
 java:
   applicationPort: 8485
+  aadIdentityName: send-letter
   keyVaults:
     rpe-send-letter:
-      resourceGroup: rpe-send-letter-service
       secrets:
         - ftp-user
         - ftp-private-key
         - ftp-public-key
+        - upload-summary-report-recipients
+        - send-letter-service-POSTGRES-PASS
+        - encryption-public-key
+        - app-insights-instrumentation-key
+        - reports-email-password
+        - reports-email-username
   environment:
     # db
-    LETTER_TRACKING_DB_HOST: "{{ .Release.Name }}-postgresql"
-    LETTER_TRACKING_DB_NAME: "{{ .Values.postgresql.postgresqlDatabase}}"
-    LETTER_TRACKING_DB_USER_NAME: "{{ .Values.postgresql.postgresqlUsername}}"
-    LETTER_TRACKING_DB_PASSWORD: "{{ .Values.postgresql.postgresqlPassword}}"
+    LETTER_TRACKING_DB_HOST: "rpe-send-letter-service-db-{{ .Values.global.environment }}.postgres.database.azure.com"
+    LETTER_TRACKING_DB_PORT: "5432"
+    LETTER_TRACKING_DB_NAME: "send_letter"
+    LETTER_TRACKING_DB_USER_NAME: "send_letter@rpe-send-letter-service-db-{{ .Values.global.environment }}"
+    LETTER_TRACKING_DB_CONN_OPTIONS: "?sslmode=require"
+
     # db - migration
-    FLYWAY_URL: "jdbc:postgresql://{{ .Release.Name }}-postgresql:5432/{{ .Values.postgresql.postgresqlDatabase}}"
-    FLYWAY_NOOP_STRATEGY: "false"
+    FLYWAY_URL: "jdbc:postgresql://rpe-send-letter-service-db-{{ .Values.global.environment }}.postgres.database.azure.com:5432/send_letter?sslmode=require"
+
     # encryption
-    ENCRYPTION_ENABLED: "false"
-    ENCRYPTION_PUBLIC_KEY: ""
+    ENCRYPTION_ENABLED: "true"
+
     # schedules
     SCHEDULING_ENABLED: "true"
+    SCHEDULING_LOCK_AT_MOST_FOR: "PT10M"
+
     # inherited
-    S2S_NAME: "send_letter_tests"
-    S2S_URL: "http://rpe-service-auth-provider-aat.service.core-compute-aat.internal"
+    S2S_URL: "http://rpe-service-auth-provider-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal"
+
+    FILE_CLEANUP_ENABLED: "true"
+    FILE_CLEANUP_CRON: "0 15 * * * *"
+
     FTP_HOSTNAME: "cmseft.services.xerox.com"
     FTP_PORT: "22"
     FTP_FINGERPRINT: "SHA256:3tX3DIkqd1Loz2alHfnt+qjHocfxk0YUOZHlnf9Zgdk"
     FTP_TARGET_FOLDER: "TO_XEROX"
     FTP_REPORTS_FOLDER: "FROM_XEROX"
+    FTP_SMOKE_TEST_TARGET_FOLDER: "SMOKE_TEST"
+    FTP_REPORTS_CRON: "0 30 * * * *"
+    #For prod overwrite these 2 values in flux
     FTP_DOWNTIME_FROM: "23:58"
     FTP_DOWNTIME_TO: "23:59"
-    # mails
-    SPRING_MAIL_HOST: "false"
+
+    #smtp - For prod turn it on and assign values to vault reports-email-username and password
+    SMTP_HOST:  "false"
+
+    #reports. depends on smtp - For prod overwrite
+    UPLOAD_SUMMARY_REPORT_CRON: "0 0 19 * * *"
```

Comment:
is this needed?

Reply:
we can probably ignore it (remove from here). same with `FILE_CLEANUP_CRON`